### DISCUSSION TITLE
Can we use laravel/ui in laravel 8 as like in laravel 7.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [laravel/framework#34214] [General] [answered]
I have a project which is developed in laravel 7 and i want to upgrade into laravel 8. But after the release with the new version the laravel/ui is removed instead of jetstream. Is it possible to used laravel/ui in laravel 8 ?

**Comment by driesvints** [comment, marked as answer]
We've updated it so people can upgrade but for new projects we recommend using Jetstream.

**Reply by shankhadevpadam** [reply]
Ok thanks.

**Comment by browner12** [comment]
So we are forced to use Jetstream now? I feel like we should pull some of the authentication files back into core so people have the tools to build their auth easily if they want to.

**Reply by driesvints** [reply]
You aren't forced to do anything. The current laravel/ui will continue to work as is if you upgrade to Laravel 8.

**Reply by browner12** [reply]
got it, thanks! was confused by the message on `laravel/ui`

**Reply by CyferZ** [reply]
Will jetstream and laravel/ui clash if both have been installed with an auth ui? Asking more out of curiosity in case I would like to use Bootstrap, or Halfmoon

**Reply by driesvints** [reply]
You can't use both jetstream and ui.
